lib/resty/core/regex.lua

```diff
@@ -154,6 +155,19 @@ local function get_max_regex_cache_size()
 end
 
 
+local regex_cache_is_empty = true
+
+function _M.is_regex_cache_empty()
```

Comment:
We don't need this global function. Just reference the top-level local variable wherever needed.